lib/beaker/host/pswindows.rb

```diff
@@ -23,5 +23,17 @@ def platform_defaults
       })
     end
 
+    attr_reader :network_separator, :external_copy_base, :system_temp_path
+    def initialize name, host_hash, options
+      super
+
+      @network_separator = '/'
+      @external_copy_base = '/programdata'
```

Comment:
I tried this, which returns `C:\PROGRA~3`, which doesn't work with scp.  I got it to present the directory to SCP just like having the value hard-coded does, and it would still fail, saying that the files don't exist:

      localhost $ scp ../my_beaker_files/tickets/bkr604/jenkins1.yml win-2008r2-x86_64-agent:/PROGRA~3 {:ignore => }
      RuntimeError: scp: /PROGRA\~3: Error opening file: No such file or directory.

I can login and verify that the files do exist after the run, and create the files before the SCP, and I still get this error message.

I'd rather keep what's there now and works, and create another ticket to improve this afterwards.